evap/staff/importers.py

```diff
@@ -141,14 +145,11 @@ def check_column_count(self, expected_column_count):
             if sheet.ncols != expected_column_count:
                 self.errors.append(_("Wrong number of columns in sheet '{}'. Expected: {}, actual: {}").format(sheet.name, expected_column_count, sheet.ncols))
 
-    def for_each_row_in_excel_file_do(self, parse_row_function):
+    def for_each_row_in_excel_file_do(self, row_function):
         for sheet in self.book.sheets():
             try:
                 for row in range(self.skip_first_n_rows, sheet.nrows):
-                    line_data = parse_row_function(sheet.row_values(row))
-                    # store data objects together with the data source location for problem tracking
-                    self.associations[(sheet.name, row)] = line_data
-
+                    row_function(sheet.row_values(row), sheet, row)
```

Comment:
I could reverse this change in interface at the moment, it is not needed with the current implementation.

Reply:
I don't mind either way.

Reply:
ok, it's your choice then, @felixrindt

Reply:
I'm gonna reverse it then to save on an argument...

Reply:
then lets keep it